src/main/java/net/minecraftforge/fmp/client/multipart/IFastMSRPart.java

```diff
@@ -0,0 +1,17 @@
+package net.minecraftforge.fmp.client.multipart;
+
+import net.minecraftforge.fmp.multipart.IMultipart;
+
+/**
+ * Interface that adds {@link FastMSR} support to {@link IMultipart}.<br/>
+ */
+public interface IFastMSRPart extends IMultipart
+{
+    
+    /**
+     * Checks whether or not this part currently needs a {@link FastMSR} or it should just use a
+     * {@link MultipartSpecialRenderer}.
+     */
+    public boolean hasFastRenderer();
```

Comment:
This was indeed left over from the 1.9 version when I finally figured out how to do FastTESRs for multiparts. I'll throw it into IMultipart.